onboard/rocks_catalog_actions.go

```diff
@@ -84,10 +93,15 @@ func (c *RocksCatalogRepoActions) Commit(ctx context.Context, commitMsg string,
 	if c.createdMetaRangeID == nil {
 		return "", ErrNoMetaRange
 	}
-	commitID, err := c.metaRanger.CommitExistingMetaRange(ctx, c.repoID, catalog.DefaultImportBranchName, *c.createdMetaRangeID, c.committer, commitMsg, graveler.Metadata(metadata))
+	commitID, err := c.metaRanger.CommitExistingMetaRange(ctx, c.repoID, c.commitID, *c.createdMetaRangeID, c.committer, commitMsg, graveler.Metadata(metadata))
```

Comment:
One thing related to this flow - in case we manage to complete everything until the point we try to update the branch and fail.
Retry the operation, will it add the same commit again? because the ref manager just AddCommit - maybe we need to ident.ContentAddress(commit) and skip this step in case the commit is already there, so retry will work.

Reply:
Added validation.